src/dataconnect/build.ts

```diff
@@ -1,15 +1,80 @@
 import { DataConnectEmulator } from "../emulator/dataconnectEmulator";
 import { Options } from "../options";
 import { FirebaseError } from "../error";
-import { prettify } from "./graphqlError";
+import * as experiments from "../experiments";
+import { promptOnce } from "../prompt";
+import * as utils from "../utils";
+import { prettify, prettifyWithWorkaround } from "./graphqlError";
 import { DeploymentMetadata } from "./types";
 
-export async function build(options: Options, configDir: string): Promise<DeploymentMetadata> {
-  const buildResult = await DataConnectEmulator.build({ configDir });
+export async function build(
+  options: Options,
+  configDir: string,
+  dryRun?: boolean,
+): Promise<DeploymentMetadata> {
+  const args: { configDir: string; projectId?: string } = { configDir };
+  if (experiments.isEnabled("fdcconnectorevolution") && options.projectId) {
+    process.env["DATA_CONNECT_PREVIEW"] = "conn_evolution";
```

Comment:
Can the emulator take multiple preview flags at once? If so, this should probably append the flag to whatever the current env var is